lib/vdsm/storage/mpathconf.py

```diff
@@ -85,9 +86,11 @@ def format_blacklist(wwids):
 
     Returns:
         A formatted blacklist section string.
+        RBD devices were added to the multipath blacklist (BZ#1881832)
```

Comment:
This is not the place to document why RBD is included.

Reply:
Mentioned it on multipath.py, is there a better place/phrasing?